Sources/Sentry/SentryFramesTracker.m

```diff
@@ -215,9 +203,9 @@ - (void)recordTimestamp:(uint64_t)timestamp value:(NSNumber *)value array:(NSMut
 
 - (SentryScreenFrames *)currentFrames
 {
-    NSUInteger total = atomic_load_explicit(&_totalFrames, SentryFramesMemoryOrder);
-    NSUInteger slow = atomic_load_explicit(&_slowFrames, SentryFramesMemoryOrder);
-    NSUInteger frozen = atomic_load_explicit(&_frozenFrames, SentryFramesMemoryOrder);
+    NSUInteger total = _totalFrames;
+    NSUInteger slow = _slowFrames;
+    NSUInteger frozen = _frozenFrames;
```

Comment:
`l`: I think we could directly pass these values to the init of `SentryScreenFrames`. No need to create local variables.